projects/core/src/lib/operators/dialogs/column-range-filter/column-range-filter.component.ts

```diff
@@ -24,31 +45,40 @@ export class ColumnRangeFilterComponent implements OnInit, OnDestroy {
 
     attributes$ = new ReplaySubject<Array<string>>(1);
 
-    form: UntypedFormGroup;
+    form: FormGroup<ColumnRangeFilterForm>;
+
```

Comment:
When you add a filter for two attributes where both attributes are numerical, you have both histograms.

Then, when you remove the **first** filter, the second filter seems to become the first (and only) one.

However, the histogram is gone.